content/en/data_jobs/databricks.md

```diff
@@ -43,7 +40,22 @@ You can choose to install the Agent globally, or on a specific Databricks cluste
 1. In Databricks, click your display name (email address) in the upper right corner of the page.
 1. Select **Admin Settings** and click the **Compute** tab.
 1. In the **All purpose clusters** section, next to **Global init scripts**, click **Manage**.
-1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the init script.
+1. Click **Add**. Name your script. Then, in the **Script** field, copy and paste the following script, remembering to replace the placeholders with your parameter values. 
+
+   ```bash
+   #!/bin/bash
+
+   # Required parameters
+   export DD_API_KEY=<YOUR API KEY>
+   export DD_SITE=<YOUR DATADOG SITE>
+   export DATABRICKS_WORKSPACE=<YOUR WORKSPACE NAME> 
+
+   # Download and run the latest init script
+   bash -c "$(curl -L https://dd-data-jobs-monitoring-setup.s3.amazonaws.com/scripts/databricks/databricks_init_latest.sh)" || true
+   ```
+
+   The script above sets the required parameters, downloads and runs the latest init script for Data Jobs Monitoring in Databricks. If you want to pin your script to a specific version, you can replace the file name in the URL with `databricks_init_1.0.0.sh` to use the last stable version.
```

Comment:
@yiliangzhou it will probably be annoying to update the docs everytime we have a new stable version, so is there somewhere we can link to where the user can find this as we add more stable releases over time?

Reply:
Agree it will be annoying and something to keep in mind, but I think it is acceptable to have to manually change it for now

Reply:
Same, this is fine for now until we have have a public repo.